Misc/NEWS.d/next/Library/2020-05-27-00-09-52.bpo-16995.4niOT7.rst

```diff
@@ -0,0 +1 @@
+Add Base32 w/ Extended Hex Alphabet functions to the base64 module (b32hexencode, b32hexdecode)
```

Comment:
```suggestion
Add :func:`base64.b32hexencode` and :func:`base64.b32hexdecode` to support Base 32 Encoding with Extended Hex Alphabet.
```

Reply:
If you'd like credit in [the changelog](https://docs.python.org/3/whatsnew/changelog.html), you can also add your name to this, like:

`Patch contributed by Filipe Laíns.`

Also, is this based on Matthäus Wander's patch? If so we should probably give them attribution as well, like:

`Patch contributed by Matthäus Wander and Filipe Laíns.`

This should probably also have a What's New entry.

Reply:
No need for credit and the patch was made from scratch.